File: Feitos em C Sharp/2862.cs
using System;
namespace ConsoleApp15 {
    class Program {
        static void Main(string[] args) {
            int a = int.Parse(Console.ReadLine());
            for (int x = 0; x < a; x++) {
                int b = int.Parse(Console.ReadLine());
                if (b > 8000) {
                    Console.WriteLine("Mais de 8000!");
                }
                else {
                    Console.WriteLine("Inseto!");
                }
            }
        }
    }
}

File: Feitos em C/2862.c
#include <stdio.h>
int main() {
	int a,x;
	scanf("%d",&a);
    for(x=0;x<a;x++){
    	int b;
    	scanf("%d",&b);
    	if(b>8000){
    		printf("Mais de 8000!\n");
		}
		else{
			printf("Inseto!\n");
		}
	}
    return 0;
}
